esp-hal/src/rtc_cntl/rtc/esp32s3.rs

```diff
@@ -39,6 +39,7 @@ pub(crate) fn configure_clock() {
 // System Reset: Reset the whole digital system, including RTC sub-system
 // Chip Reset:   Reset the whole chip, including the analog part
 
+/// SOC reset reason
```

Comment:
This is the third variation of this comment :D `SOC Reset Reason.` vs `SOC Reset Reason` vs `SOC reset reason`. Pick one, please.